### DISCUSSION TITLE
Is there a way to have multiple instance of the same app having its own tray icon?

### DISCUSSION THREAD

**Discussion by bagusnl** [HavenDV/H.NotifyIcon#146] [Q&A] [answered]
Hi, 
I'm trying to implement NotifyIcon in my WinUI3 application using the example on readme file. I'm currently having problem where if multiple instance of the same app is running, taskbar icon from older instance gets disposed and replaced by the one from new instance (seen in the video with `- #2` prefix.
Can the taskbar icon created by the 2nd instance gets its own icon instead of replacing the old one?

References:
- https://github.com/CollapseLauncher/Collapse/blob/main/CollapseLauncher/XAMLs/MainApp/TrayIcon.xaml
- https://github.com/CollapseLauncher/Collapse/blob/main/CollapseLauncher/XAMLs/MainApp/TrayIcon.xaml.cs
- Where it gets loaded 
https://github.com/CollapseLauncher/Collapse/blob/5a6dfd1ecfd45091d1f8d2750ebf26d13d42aaf2/CollapseLauncher/XAMLs/MainApp/MainWindow.xaml#L138

https://github.com/HavenDV/H.NotifyIcon/assets/28079733/a4a8a2df-6daf-4cd0-b6b3-aab11310de7a

Thank you!

Edit:

I want to add a note that if the executable is from different folder, it works. The behavior happen if the same executable is run twice
![image](https://github.com/HavenDV/H.NotifyIcon/assets/28079733/f2569bc6-d158-4699-aaf4-dc5e200cc84c)
![image](https://github.com/HavenDV/H.NotifyIcon/assets/28079733/7308925a-9035-4ae5-af73-a2ac6ad50b04)

**Comment by HavenDV** [comment, marked as answer]
Hello. You can use these properties to allow your application have multiple icons:
![image](https://github.com/HavenDV/H.NotifyIcon/assets/3002068/3bc93376-c495-49ca-96cf-3a08e5c6b329)

**Reply by bagusnl** [reply]
Thank you for the help, sorry for taking awhile for closing the thread
